dist/listbox/ds6/listbox.css

```diff
@@ -1,62 +0,0 @@
-.listbox {
```

Comment:
Yep. This one can be deleted as we never really had a DS6 listbox.

Reply:
👍